README.Rmd

```diff
@@ -131,13 +131,14 @@ Each `xportr_` function has been written in a way to take in a part of the speci
 
 ```{r, warning = FALSE, message=FALSE, eval=TRUE}
 adsl %>%
-  xportr_type(var_spec, "ADSL", verbose = "warn") %>%
-  xportr_length(var_spec, "ADSL", verbose = "warn") %>%
-  xportr_label(var_spec, "ADSL", verbose = "warn") %>%
-  xportr_order(var_spec, "ADSL", verbose = "warn") %>%
-  xportr_format(var_spec, "ADSL") %>%
-  xportr_df_label(dataset_spec, "ADSL") %>%
-  xportr_write("adsl.xpt")
+  xportr_domain_name("ADSL") %>%
+  xportr_type(var_spec, verbose = "warn") %>%
+  xportr_length(var_spec, verbose = "warn") %>%
+  xportr_label(var_spec, verbose = "warn") %>%
+  xportr_order(var_spec, verbose = "warn") %>%
+  xportr_format(var_spec) %>%
+    xportr_df_label(dataset_spec, "ADSL") %>%
+  xportr_write("adsl.xpt", label = "Subject-Level Analysis Dataset")
```

Comment:
If we are using `xportr_df_label` do we need the additional label call in `xportr_write()`?